### PR TITLE
Removed mentions of psij-ci-load since it has been phased out by

### PR DESCRIPTION
**Pull request by hategan**
psij-ci-env. Psij-ci-run still attempts to load it if it's there just in case.

### CODE REVIEW

File: psij-ci-run [hunk 1]
```diff
@@ -16,7 +16,10 @@ failifempty() {
     fi
 }
 
-source psij-ci-load
+if [ -f psij-ci-load ]; then
+    source psij-ci-load
+fi
```
Comment (jameshcorbett) [inline psij-ci-run:21]: I'm not completely sold on keeping the invocation of `psij-ci-load`. Any user who put anything in their psij-ci-load is going to get a conflict when they pull this commit (I think) and so should figure out that something has changed. How about putting a warning message instead? Or keeping what you have but also echoing a deprecation warning.

**Review by jameshcorbett — APPROVED** [review]
Thanks @hategan , LGTM. One minor comment below.

File: psij-ci-run (hunk 1 above)
Comment (hategan) [reply psij-ci-run:21]: I'm thinking that any user who has a working setup with something in `psij-ci-load` can and should probably keep it as is.

That said, I'm not completely sold on keeping it either, but for a slightly different reason: if we need to instruct a user to do some fancy setup requiring something like `psij-ci-load`, we could as well instruct them to put the relevant lines in `psij-ci-run`.

Comment (hategan) [reply psij-ci-run:21]: So maybe there's consensus here that this should be removed.

**Pull request merged**